fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckAction.kt

```diff
@@ -5,6 +5,7 @@
 package org.mozilla.fenix.shopping.store
 
 import mozilla.components.lib.state.Action
+import org.mozilla.fenix.shopping.middleware.ReviewQualityCheckNavigationMiddleware
```

Comment:
Let's not make the action depend on the middleware, we can simplify by creating different actions for different link clicks. We can then add all the link urls as constants in nav middleware which just reacts to those actions.